examples/passive_scalar/benchmark/gaussian2d/seeder.lua

```diff
@@ -0,0 +1,95 @@
+-- draw mesh for a 2d cube
+
+require 'args'
+
+folder    = 'mesh/'
+comment   = "gaussian_pulse"
+
+minlevel  = level_usr
+print("level of treelm: ", minlevel)
+
+bc_origin = {-nelem-2.5, -nelem-2.5, -0.5 - 4}
+length_bnd = length_usr
+
+bounding_cube = { origin = bc_origin,
+                  length = length_bnd }
+
+ebug = {debugMode = true, debugFiles = false, debugMesh='debug/' }
```

Comment:
I know this is done in several places to "comment out" settings. But I think it is clearer to prepend a `NO` in front of the renamed variable, like `NOdebug`.